packages/vuetify/src/components/VStepper/__tests__/__snapshots__/VStepperStep.spec.ts.snap

```diff
@@ -1,12 +1,13 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`VStepperStep.js should accept a custom color 1`] = `
+exports[`VStepperStep.ts should accept a custom color 1`] = `
 
 <div class="v-stepper__step v-stepper__step--inactive v-stepper__step--complete">
   <span class="v-stepper__step__step pink">
     <i aria-hidden="true"
-       class="v-icon mdi mdi-check theme--light"
+       class="v-icon material-icons theme--light"
     >
+      $vuetify.icons.complete
```

Comment:
You need to mock $vuetify object, if you need help, PM me in Discord, I'll do this

Reply:
P.S.: not a joke :smiley:

Reply:
Updated